testconvolute.m

```diff
@@ -2,42 +2,52 @@
 ht = 0.01:0.01:2;%function h(t)
 Zt = convolute(xt,ht);%function Z(t), convoluted function
 cCheck = conv(0.01*xt,ht);%Our check for our test
-Zt %comment out if you want to print @MickeyDang
-cCheck %comment out if you want to print @MickeyDang
-T1 = false; %Bool for Test 1
-if is_appEq(Zt,cCheck) %Easier to set a tolerance because MATLAB has negative 0's
-    fprintf("Test1 has passed. Convolution matches with conv(xt,ht)");
-    T1 = true;
-end
+Zt; %comment out if you want to print @MickeyDang
+cCheck; %comment out if you want to print @MickeyDang
+T = false(2,1);
+T(1,1) = test_convolute(xt,ht,"1"); %Bool for Test 1
+
 
 %Run Test two 
-t = 0.01:0.01:2
+t = 0.01:0.01:2;
 sint = sin(2*pi*t);
 cost = cos(2*pi*t);
-Z2 = convolute(sint,cost);
-cCheck2 = conv(.01*sint,cost);
-Z2 %comment out if you want to print @MickeyDang
-cCheck2 %comment out if you want to print @MickeyDang
-T2 = false; %Bool for Test two
-if is_appEq(Z2,cCheck2)
-    fprintf("Test2 has passed. Convolution matches with conv(xt,ht)\n");
-    T2 = true;
-end
+T(2,1) = test_convolute(xt,ht,"2"); %Bool for Test two
+
 %Load Summary
-fprintf("Summary of Test Results:\n")
-if T1
-    fprintf("Test1: Success\n")
-else
-    fprintf("Test1: Failed\n")
+load_summary(T);
+
+function load_summary(T)
+fprintf("Summary of results:\n");
+    for i = 1:length(T)
+        if T
+            fprintf("Test " + num2str(i) + " Succeeded\n");
+        else
+            fprintf("Test " + num2str(i) + " Failed\n");
+        end
+    end
 end
-if T2
-    fprintf("Test2: Success\n")
-else
-    fprintf("Test2: Failed\n")
+function check = test_convolute(xt,ht, test_name)
+    check = is_appEq(conv(0.01*xt,ht),convolute(xt,ht));
+    if check
+        fprintf("Test " + test_name + " Succeeded\n")
+    else
+        fprintf("Test " + test_name + " Failed\n")
+    end
+end
+
+function eq = is_appEq(arr1,arr2); %function to check if arr1 close toarr2

```

Comment:
can we rename it to `isApproxEqual()` rather than having a comment to an abbreviated name?